Package.resolved

```diff
@@ -255,10 +255,10 @@
       },
       {
         "package": "SwiftPM",
-        "repositoryURL": "https://github.com/apple/swift-package-manager.git",
+        "repositoryURL": "https://github.com/finestructure/swift-package-manager.git",
         "state": {
-          "branch": "swift-DEVELOPMENT-SNAPSHOT-2021-05-04-a",
-          "revision": "0c9c8d55889405f39b093ba4868db375b5b7e16c",
+          "branch": "signer-default-observability-scope",
+          "revision": "25df5347eef59c5f0ae86f5e4ee7723ff4eb78f7",
```

Comment:
My branch contains a workaround for accessing `PackageCollectionSigning`. I've opened a PR against SPM here: https://github.com/apple/swift-package-manager/pull/3836